Makefile

```diff
@@ -124,16 +127,17 @@ install-go-ci-dependencies:
 	go get -u golang.org/x/lint/golint
 
 compile-protos-go:
-	$(foreach dir,types serving core storage,cd ${ROOT_DIR}/protos; protoc -I/usr/local/include -I. --go_out=plugins=grpc,paths=source_relative:../sdk/go/protos feast/$(dir)/*.proto;)
+	mkdir -p ./go/protos
+	$(foreach dir,$(PROTO_TYPE_SUBDIRS),cd ${ROOT_DIR}/protos; protoc -I/usr/local/include -I. --go-grpc_out=paths=source_relative:../go/protos --go_out=paths=source_relative:../go/protos feast/$(dir)/*.proto;)
 
 test-go:
-	cd ${ROOT_DIR}/sdk/go; go test ./...
+	go test ./...
 
 format-go:
-	cd ${ROOT_DIR}/sdk/go; gofmt -s -w *.go
+	gofmt -s -w go/**/*.go
 
 lint-go:
-	cd ${ROOT_DIR}/sdk/go; go vet
+	go vet ./go/feast ./go/server
```

Comment:
Adding a note so that we don't forget: Need to add lint-go to pre-commit hook

Reply:
any reason we're not using `gofmt` here?